scripts/mysql_slave_lag.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+seconds=`mysql -s -e 'show slave status\G' | grep Seconds_Behind_Master | awk {'print $2'}`
+test "$seconds" = "0"
```

Comment:
Given that this tool is only run in an emergency I feel alerting on >0 is sane behaviour.